package/src/frontend/browser/Agent.js

```diff
@@ -168,6 +171,16 @@ export const Agent = function(userAgent) {
     var match = userAgent.match(exp);
     return match && match[1] && parseInt(match[1], 10) >= version;   
   }
+
+  //After ios13 update, iPad reports the same user string
+  //as Safari on Dekstop MacOS.
+  //At the time of this writing there are no other devices
+  //with multi-touch support other than IOS/iPadOS
+  //See: https://stackoverflow.com/a/58064481
+  function matchesiPadSafari13AndAbove() {
+    return navigator.maxTouchPoints > 1 &&
+            navigator.platform === 'MacIntel';       
+  }
```

Comment:
Let's also check the user agent string claims to be a Safari. Otherwise any browser on Mac will become an iPad soon as there is a Mac with touch input, right?

Reply:
We want to check that above in matchesMobileSafari(). Right? Any other browser will return true for matchesMobileSafari() on IOS.

Reply:
> Any other browser will return true for matchesMobileSafari() on IOS.

I don't understand.

Reply:
I think i misunderstood, i will add the check for Safari here.